text/judgements.tex

```diff
@@ -12,70 +12,92 @@ \section{Judgements}\label{sec:judgements}
 
 \subsection{State}
 
-The judgements state includes three items, an allow-set ($\psi_\mathbf{a}$), a ban-set ($\psi_\mathbf{b}$) and a punish-set ($\psi_\mathbf{p}$). The allow-set contains the hashes of all work-reports which were disputed and judged to be accurate. The ban-set contains the hashes of all work-reports which were disputed and whose accuracy could not be confidently confirmed. The punish-set is a set of Ed25519 keys representing validators which were found to have guaranteed a report which was confidently found to be invalid.
+The judgements state includes three items, an allow-set ($\psi_\mathbf{a}$), a ban-set ($\psi_\mathbf{b}$), a corrupt-set ($\psi_\mathbf{c}$) and a punish-set ($\psi_\mathbf{p}$). The allow-set, ban-set and corrupt-set contains the hashes of all work-reports which were respectively judged to be correct, uncertain or incorrect. The punish-set is a set of Ed25519 keys representing validators which were found to have misjudged a work-report.
```

Comment:
So the punish set now includes evryone who misjudged and not just backers. Good, because all these validators should not be guarentors.

The staking chain will want to make a distinction to slash guarantors and others by different amounts, which liekely means that we have to repost the judgement on it, rather than relying on JAM's punish set. Tis is workable and maybe there are judgements that only go to that chain.